__tests__/app.test.js

```diff
@@ -78,3 +78,42 @@ describe("GET - status:404, bad URL request", () => {
       });
   });
 });
+
+describe("GET /reviews/:review_id", () => {
+  test("GET - status:200, responds with a review object", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(review).toBeInstanceOf(Object);
+        expect.objectContaining({
+          review_id: expect.any(Number),

```

Comment:
Are we expecting review_id to be any number on this object? Just make sure your tests are asserting the information that you're looking for. Here, it should be an ID of 1